xds/src/main/java/io/grpc/xds/PriorityLoadBalancer.java

```diff
@@ -153,7 +153,9 @@ private void tryNextPriority(boolean reportConnecting) {
     }
     // TODO(zdapeng): Include error details of each priority.
     logger.log(XdsLogLevel.DEBUG, "All priority failed");
-    updateOverallState(TRANSIENT_FAILURE, new ErrorPicker(Status.UNAVAILABLE));
+    String lastPriority = priorityNames.get(priorityNames.size() - 1);
```

Comment:
I don't think using the last priority's error message is better than just saying cannot establish connection to any priority.

Reply:
I am fine with using the last priority's error state as the overall error state for the LB. I am encountering the similar situation in cluster_resolver LB.